tests/features/steps/qc_e2e.py

```diff
@@ -0,0 +1,157 @@
+        case "Packstack":
+            qc_form.packstack_rdbox.click()
+
+            
+    time.sleep(3)
+
+    qc_form.qcnext_btn.click()
+    time.sleep(3) 
+
+    qc_form.regionrdu_rdbox.click()  #create class to select the right option based on the .json file
+    time.sleep(2)
+
+    qc_form.qcnext_btn.click()
+    time.sleep(3)
+    
+    qc_form.input(qc_form.clusterid_fld, dt_request.get_data("cluster_id"))
+
+    qc_form.baseosimage_slct.click()
+    time.sleep(2)
+
+    qc_form.qcform_body.click()
+
+    image = Select(qc_form.baseosimage_slct)
+    image.select_by_visible_text(dt_request.get_data("image_select")) 
+    
+    time.sleep(2)
+
+    qc_form.qcform_body.click()
+
+    time.sleep(2)
+
+    qc_form.nodecount_slct.send_keys((Keys.CONTROL, "a"), dt_request.get_data("node_count"))
+    
+    nd_flavor = Select(qc_form.nodeflavor_slct)
+    nd_flavor.select_by_value(dt_request.get_data("node_flavor"))
+
+    qc_form.input(qc_form.aditionalhd_slct, dt_request.get_data("aditionalvhd"))
+    qc_form.input(qc_form.aditionalhdsize_slct, dt_request.get_data("aditionalvhdsz"))
+    
+    reser_days = Select(qc_form.reservationdays_slct)
+    reser_days.select_by_value(dt_request.get_data("reservationdz"))
+    
+    qc_form.qcnext_btn.click()
+
+    time.sleep(4)
+
+    yum_update = Select(qc_form.yumupdate_slct) 
+    yum_update.select_by_visible_text(dt_request.get_data("yumupdate"))
+
+    time.sleep(2)
+
+    reg_method = Select(qc_form.registrationmethod_slct)
+    reg_method.select_by_visible_text(dt_request.get_data("reg_method"))
+
+    time.sleep(2)
+    
+    qc_form.qcnext_btn.click()
+
+    time.sleep(5)
+
+    assert qc_form.product_txt.text == dt_request.get_data("prod_select")
+    assert qc_form.region_txt.text == dt_request.get_data("region_select")
+    
+    exp_days = dt_request.get_data("reservationdz")+" days"
+    assert qc_form.reservexpiration_txt.text == exp_days
+
+    assert dt_request.get_data("node_count") in qc_form.nodecount_txt.text
+    assert qc_form.nodeflavor_txt.text == dt_request.get_data("node_flavor")
+    assert qc_form.nbaddtvdk_txt.text == dt_request.get_data("aditionalvhd")
+    assert qc_form.szaddtvdk_txt.text == dt_request.get_data("aditionalvhdsz")
+    assert qc_form.execyumupdt_txt.text == dt_request.get_data("yumupdate")
+    assert qc_form.regsrc_txt.text == dt_request.get_data("reg_method")
+
+    qc_form.finish_btn.click()
+
+    time.sleep(6)
+
+@then(u'the cluster must be provisioned and available at main page')
+def step_impl(context):
+    dt_request = GetAttribute()
+   
+    cluster_name = dt_request.get_data("cluster_id")
+ 
+    for i in range (2, 100, +1):
+        pos_ph = str(i)
+        path_table_name = context.browser.find_element(By.XPATH, '/html/body/div/div/main/section/article/div[2]/table/tbody['+pos_ph+']/tr[1]/td[2]/a')
+        name_in_table = path_table_name.text
+        if (name_in_table == cluster_name):
+            assert True
```

Comment:
Not sure, but I think that `assert True` doesn't really do anything. Also if the cluster name wasn't found in the table there would be no error thrown and the test would be considered "successful". I think we can remove the `assert True` line and use the [for/else clause](https://book.pythontips.com/en/latest/for_-_else.html) to test if we broke out of the loop or not and potentially fail the test.

Also we are not testing the status of the cluster, only if the name is displayed. I was able to run the test successfully but the info in the table said that cluster provisioning failed.

![image](https://user-images.githubusercontent.com/52234617/196913687-c6c71da8-e51d-4a9a-a458-53c1ee29a5eb.png)